packages/xo-server/src/xapi/mixins/vm.js

```diff
@@ -82,7 +84,22 @@ export default {
     )
     $defer.onFailure(() => this.deleteVm(vmRef))
 
-    // TODO: copy BIOS strings?
+    // Copy BIOS strings
+    if (
+      props.hvmBootFirmware !== 'uefi' &&
+      isVmHvm(template) &&
+      copyHostBiosStrings
+    ) {
+      await this.callAsync(
+        'VM.copy_bios_strings',
+        vmRef,
+        this.getObject(
+          props.affinityHost === undefined
```

Comment:
You can use `nullish` operator. `props.affinityHost ?? this.getObject(template.$pool).master`